### DISCUSSION TITLE
Welcome to obsidian-open-gate Discussions!

### DISCUSSION THREAD

**Discussion by nguyenvanduocit** [nguyenvanduocit/obsidian-open-gate#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by seawee1** [comment]
Hi, just wanted to jump in and say thank you for this great plugin. Works really well and helps so much :)

**Comment by MatthewDanh** [comment]
I would like to say thank you as well, from a chronic task switcher. Pleasure using your well built plug-in. 🥇
